ckanext/biskit/templates/base.html

```diff
@@ -1,2 +1,6 @@
 {% resource 'biskit/styles/biskit.css' %}
 {% ckan_extends %}
+
+{% block links -%}
+<link rel="shortcut icon" href="/img/ckan.ico" />
```

Comment:
@panveel can you confirm that this works as I don't see any `ckan.ico` file within `public/img` folder or `fanstatic` folder?

Reply:
Yes, it's available.